src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_AllocatePrisonOffenderManagerTest.java

```diff
@@ -446,6 +446,48 @@ void willReturnAConflictWhenSupplyingANOMSNumberWithDuplicateActiveOffenders() {
                 .statusCode(409);
     }
 
+    //TODO - Enable this test when EMAIL_ADDRESS and TELEPHONE_NUMBER columns are added in production
+    //@Test
```

Comment:
Vey minor, I think `@Disabled`  might have been better so you can give a nice description why is not run

Reply:
Done